### PR TITLE
refactored update user endpoint

### PR DESCRIPTION
**Pull request by mikey247**
**REFACTORED THE UPDATE USER ENDPOINT SO THAT THE EMAIL FIELD IS NOT REQUIRED**
**BEARER TOKEN IS STILL REQUIRED**

**LINEAR TICKET**
https://linear.app/team-discripto/issue/BE-54/create-endpoint-to-update-user-info

### CODE REVIEW

File: internal/model/user.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ type PasswordForgot struct {
 type UpdateUser struct {
 	FirstName       string `bson:"first_name" json:"first_name"`
 	LastName        string `bson:"last_name" json:"last_name"`
-	Email           string `bson:"email" json:"email" validate:"required"`
+	Email           string `bson:"email" json:"email"`
```
Comment (kataszn) [inline internal/model/user.go:63]: we have to validate the email
is there any reason why u removed this?

File: service/user/user.go [hunk 2]
```diff
@@ -193,35 +193,48 @@ func ProfilePictureServiceUpload(userId interface{}, image io.ReadCloser, filena
 
 }
 
-func UpdateUserService(user model.UpdateUser) (int, error) {
+func UpdateUserService(user model.UpdateUser, userId interface{}) (int, error) {
+	string_id,_ := userId.(string)
+	string_id = string_id[10:len(string_id)-2]
+	fmt.Println(string_id)
```
Comment (kataszn) [inline service/user/user.go]: dont forget to remove printlns

**Review by kataszn — COMMENTED** [review]
check my comments

File: internal/model/user.go (hunk 1 above)
Comment (mikey247) [reply internal/model/user.go:63]: Yes remember I was checking with email at first, now the Token is all that's needed

**Comment by mikey247** [comment]
You've still not merged this

**Pull request merged**